pvlib/losses.py

```diff
@@ -85,3 +86,123 @@ def soiling_hsu(rainfall, cleaning_threshold, tilt, pm2_5, pm10,
     soiling_ratio = 1 - 0.3437 * erf(0.17 * accum_mass**0.8473)
 
     return soiling_ratio
+
+
+def soiling_kimber(rainfall, cleaning_threshold=6, soiling_loss_rate=0.0015,
+                   grace_period=14, max_soiling=0.3, manual_wash_dates=None,
+                   initial_soiling=0):
+    """
+    Kimber soiling model [1]_ assumes soiling builds up at a daily rate unless
```

Comment:
```suggestion
    Calculate soiling ratio with rainfall data and a daily soiling rate using 
    the Kimber soiling model [1]_.
    
    Kimber soiling model assumes soiling builds up at a daily rate unless
```
Add a 1-line explanation for the sphinx docs:
https://external-builds.readthedocs.io/html/pvlib-python/860/api.html#losses